binder/environment.yml

```diff
@@ -10,5 +10,5 @@ dependencies:
 - pandas
 - scipy
 - voila
-- voila-gridstack=0.2.0
-- jupyterlab=3
+- voila-gridstack
```

Comment:
Maybe we could remove `voila-gridstack` from the `environment.yml`?  And add `pip install .` in a `postBuild` file.

That way we could more easily test PRs on Binder.